www/js/TimelineContext.ts

```diff
@@ -135,10 +139,6 @@ export const useTimelineContext = (): ContextProps => {
     );
     setTimelineLabelMap(newTimelineLabelMap);
     setTimelineNotesMap(newTimelineNotesMap);
-    publish('applyLabelTabFilters', {
-      timelineMap,
-      timelineLabelMap: newTimelineLabelMap,
-    });
```

Comment:
Agreed that moving this into the `LabelTab` allows us to keep the presentation logic largely in the label which is the right thing to do.